mods/mtg_craftguide/locale/template.txt

```diff
@@ -1,57 +1,53 @@
 # textdomain: mtg_craftguide
-
-
-### init.lua ###
-
-Any black dye=
-Any black flower=
-Any blue dye=
-Any blue flower=
-Any brown dye=
 Any coal=
-Any cyan dye=
-Any dark green dye=
-Any dark grey dye=
-Any green dye=
-Any green flower=
-Any grey dye=
-Any item belonging to the group(s): @1=
-Any kind of stone block=
-Any magenta dye=
-Any orange dye=
-Any orange flower=
-Any pink dye=
-Any red dye=
-Any red flower=
 Any sand=
+Any wool=
 Any stick=
 Any vessel=
-Any violet dye=
+Any wood planks=
+Any kind of stone block=
+Any red flower=
+Any blue flower=
+Any black flower=
+Any green flower=
+Any white flower=
+Any orange flower=
 Any violet flower=
+Any yellow flower=
+Any red dye=
+Any blue dye=
+Any cyan dye=
+Any grey dye=
+Any pink dye=
+Any black dye=
+Any brown dye=
+Any green dye=
 Any white dye=
-Any white flower=
-Any wood planks=
-Any wool=
+Any orange dye=
+Any violet dye=
 Any yellow dye=
-Any yellow flower=
-Click again to show recipes.=
-Click again to show usages.=
-Cooking time: @1=
+Any magenta dye=
+Any dark grey dye=
+Any dark green dye=
+G=
+Any item belonging to the group(s): @1=
+Unknown Item=
 Fuel=
+Usage @1 of @2=
+Recipe @1 of @2=
+Previous recipe=
+Next recipe=
+Recipe is too big to be displayed.=
+Shapeless=
+Cooking time: @1=
+Search=
+Reset=
+Previous page=
 # Label for group ingredients
```

Comment:
Misplaced comment. Also in the other files

Reply:
Still here. @Wuzzy2 did you want to remove this?